ts/client/src/client.ts

```diff
@@ -4339,9 +4325,11 @@ export class MangoClient {
           1) *
         100;
     }
-    const maxPricePremiumRate =
+    let maxPricePremiumRate =
       maxPricePremiumPercent > 0 ? maxPricePremiumPercent / 100 : 0.03;
 
+    maxPricePremiumRate = maxPricePremiumRate * 2;
```

Comment:
Maybe make a separate helper for the 60s duration, 2x premium rate thing; I don't think this should be in this function